ui/apps/platform/src/Containers/Violations/Details/Deployment/SecurityContext.tsx

```diff
@@ -9,46 +9,44 @@ export type SecurityContextProps = {
 };
 
 function SecurityContext({ deployment }: SecurityContextProps): ReactElement {
-    const securityContextContainers =
-        deployment?.containers?.filter(
-            (container) =>
-                !!(
-                    container?.securityContext?.privileged ||
-                    container?.securityContext?.addCapabilities.length > 0 ||
-                    container?.securityContext?.dropCapabilities.length > 0
-                )
-        ) ?? [];
+    let content: JSX.Element[] | string = 'None';
+
+    if (deployment === null) {
+        content =
+            "Security context is unavailable because the alert's deployment no longer exists.";
+    } else {
```

Comment:
Nit: Same comment with `else if`

Reply:
This one we should keep as is. The else has a variable that needs to be assigned a value and checked in the other if statement. I could move it up so we can convert it to an `else if` but if `deployment` is null, that expression doesn't make much sense. We do have optional chaining so we can short circuit, but better to keep it farther below imo.